### PR TITLE
Added clarity to spec sheet

### PR DESCRIPTION
**Pull request by owalk**

### CODE REVIEW

**Comment by owalk** [comment]
As a spec sheet change, we should agree on the definitions before pushing them up. Let me know if there's anything that should change. With that said this is more a clarity change rather than a functional change to the spec sheet.

File: docs/specSheet.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Lunk negative interactions:
 
 
 ## Player Movement
-Lunk always goes forward, but he can choose to turn to the left of the right.
+Lunk always goes forward, but he can choose to turn to the left or the right.
+
+This means movement is rotational in nature. Lunk's heading will be adjusted but not suddenly altered. The purpose of this movement system is to give Lunk the feeling of an unwieldy character who is hard to control directly. The player can steer lunk, but not command him. The same thought process was used when creating lunks abilities and enemy interactions.
```
Comment (owalk) [inline docs/specSheet.md:36]:
```suggestion
This means movement is rotational in nature. Lunk's heading will be adjusted but not suddenly altered. The purpose of this movement system is to give Lunk the feeling of an unwieldy character who is hard to control directly. The player can steer Lunk, but not command him. The 
```

**Comment by owalk** [comment]
@Sean-Nishi this is ready for review

**Review by Sean-Nishi — APPROVED** [review]
Lunks are all capitalized.

**Pull request merged**